src/main/java/io/opentracing/contrib/global/GlobalTracer.java

```diff
@@ -38,43 +38,26 @@
  * The {@linkplain GlobalTracer} will not attempt to choose between implementations:</li>
  * <li>If no single tracer service is found, the {@link io.opentracing.NoopTracer NoopTracer} will be used.</li>
  * </ol>
+ *
+ * @see io.opentracing.util.GlobalTracer
+ * @deprecated GlobalTracer is now part of OpenTracing-java
  */
-public final class GlobalTracer implements Tracer {
```

Comment:
Because in the new situation there's no way to actually _obtain_ a reference to the old `GlobalTracer` anymore. In effect, the new version of the library will 'make' people use the new version without any code changes. That's the only reason for an additional release; 'upgrade' to the new GlobalTracer just by switching from 0.1.0 to 0.1.1 without any code changes.